data/tlg0018/tlg001/tlg0018.tlg001.1st1K-eng1.xml

```diff
@@ -0,0 +1,1013 @@
+</div>
+
+<div type="textpart" subtype="section" n="22">
+<p>For the substance was of itself destitute of arrangement, of quality, of animation, of distinctive character, and full of all disorder and confusion; and it received a change and transformation to what is opposite to this condition, and most excellent, being invested with order, quality, animation, resemblance, identity, arrangement, harmony, and everything which belongs to the more excellent idea.
+<milestone unit="chapter" n="6"/></p>
+</div>
+
+<div type="textpart" subtype="section" n="23">
+<p>And God, not being urged on by any prompter (for
+<pb n="6"/>
+who else could there have been to prompt him?) but guided by his own sole will, decided that it was fitting to benefit with unlimited and abundant favours a nature which, without the divine gift, was <choice><sic>unable of</sic><corr>unable to</corr></choice> itself to partake of any good thing; but he benefits it, not according to the greatness of his own graces, for they are illimitable and eternal, but according to the power of that which is benefited to receive his graces. For the capacity of that which is created to receive benefits does not correspond to the natural power of God to confer them; since his powers are infinitely greater, and the thing created being not sufficiently powerful to receive all their greatness would have sunk under it, if he had not measured his bounty, allotting to each, in due proportion, that which was poured upon it.</p>
+</div>
+
+<div type="textpart" subtype="section" n="24">
+<p>And if any one were to desire to use more undisguised terms, he would not call the world, which is perceptible only to the intellect, any thing else but the reason of God, already occupied in the creation of the world; for neither is a city, while only perceptible to the intellect, anything else but the reason of the architect, who is already designing to build one perceptible to the external senses, on the model of that which is so only to the intellect—</p>
+</div>
+
+<div type="textpart" subtype="section" n="25">
+<p>this is the doctrine of Moses, not mine. Accordingly he, when recording the creation of man, in words which follow, asserts expressly, that he was made in the image of God—and if the image be a part of the image, then manifestly so is the entire form, namely, the whole of this world perceptible by the external senses, which is a greater imitation of the divine image than the human form is. It is manifest also, that the archetypal seal, which we call that world which is perceptible only to the intellect, must itself be the archetypal model, the idea of ideas, the Reason of God.
+<milestone unit="chapter" n="7"/></p>
```

Comment:
I would bump any milestones at the end of a section to the start of the next section (strange placement that happens in conversion)--otherwise sort of useless here